x-pack/plugins/security_solution/common/endpoint/schema/actions.test.ts

```diff
@@ -233,6 +233,50 @@ describe('actions schemas', () => {
         });
       }).not.toThrow();
     });
+
+    it('should not work with empty string in `withOutputs` list', () => {
+      expect(() => {
+        EndpointActionListRequestSchema.query.validate({
+          startDate: 'now-1d', // yesterday
+          endDate: 'now', // today
+          statuses: ['failed', 'pending', 'successful'],
+          withOutputs: '  ',
```

Comment:
Are we testing empty string or a string with only a space? Maybe you can add both test cases

Reply:
Yeah I should add a test case for empty string as well.

Reply:
done c31926fee1e7c7593620ae3ddfcf01094331bfb4